ansible.cfg

```diff
@@ -9,6 +9,7 @@ stdout_callback		= yaml
 bin_ansible_callbacks	= True
 jinja2_extensions	= jinja2.ext.do
 forks			= 10
+allow_world_readable_tmpfiles = True
```

Comment:
Wozu wird dieser Parameter gebraucht?

Reply:
Die systemli.etherpad role braucht das. Weis aber nicht mehr genau wofür. Dachte ich jedenfalls. Jetzt habe ich das nochmal durchlaufen lassen und es geht? Wahrscheinlich weil die Datei bereits da ist...
Keine Ahnung, ob wir das jetzt lassen...

Reply:
Das müsste auch auf Host-Ebene konfigurierbar sein. Global wird das ja nicht gebraucht.

Reply:
Wenn ich die Doku richtig lese kann das nur in der ansible.cfg gesetzt werden die nur einmal eingelesen wird.

> Changes can be made and used in a configuration file which will be searched for in the following order:
> 
> * ANSIBLE_CONFIG (environment variable if set)
> * ansible.cfg (in the current directory)
> * ~/.ansible.cfg (in the home directory)
> * /etc/ansible/ansible.cfg
> 
> Ansible will process the above list and use the first file found, all others are ignored.

https://docs.ansible.com/ansible/latest/reference_appendices/config.html?highlight=allow_world_readable_tmpfiles#the-configuration-file

Reply:
Richtig, dachte ich hätte es in irgendeinem Ansible Issue gelesen, dass das geht ...

Reply:
Man könnte jetzt sicher auch die Rolle forken und überarbeiten, da sowieso deprecated Sachen (with items) benutzt werden. Wäre sowieso irgendwann fällig. Aber irgendwie war es mir den Aufwand jetzt nicht wert.

Reply:
@prisma01 Hast du dir auch die anderen Galaxy Roles angeschaut? https://galaxy.ansible.com/robertdebock/etherpad sieht auf den ersten Blick brauchbar aus.

Reply:
Hm, diese hatte ich übersehen, wird auf jedenfall besser maintained. Hat aber leider komplett andere dependencies. Da müsste aber ich alles komplett umstricken. Und kann zudem weder mysql noch redis, nur dirtydb. Dann müsste man noch ne sql -> dirtydb migration machen.

Reply:
Oh, darauf hatte ich beim Suchen jetzt nicht geachtet aber eine Datenbank Konvertierung halte ich auch für unnötig zumal sich dirtydb nicht so toll anhört ...